### PR TITLE
[Feature] Add list for etc page

### PR DESCRIPTION
**Pull request by yaoandy107**
## Summary 
Add list view for feature function in the etc page

## Todos
- [x] Custom a rounded icon widget
- [x] Add etc's item model
- [x] Add list view and function list in the etc page

## Issues
none

### CODE REVIEW

File: lib/screen/etc_page.dart [hunk 1]
```diff
@@ -1,15 +1,45 @@
...
+                EtcItem etcItem = etcItems[index];
+                return InkWell(
+                  child: ListTile(
+                    contentPadding: EdgeInsets.fromLTRB(24, 4, 24, 4),
+                    leading: RoundedIcon(
+                      icon: etcItem.icon,
+                      color: etcItem.color,
+                    ),
+                    title: Text(etcItem.text,
+                        style: TextStyle(color: Colors.black87)),
```
Comment (freddy5566) [inline lib/screen/etc_page.dart]: need to be constant, if all style (in our app) is the same

Comment (yaoandy107) [reply lib/screen/etc_page.dart]: Which should be constant? I think there is only text color should be constant, is there any others?

Comment (freddy5566) [reply lib/screen/etc_page.dart]: nope, text color is the only thing need to be constant

**Comment by yaoandy107** [comment]
@jamfly 
I have refactored the code smell, waiting for you to recheck it.

**Comment by freddy5566** [comment]
@yaoandy107 

cool, everything work perfectly.

**Review by freddy5566 — APPROVED** [review]

**Pull request merged**